client/state/reader/site-blocks/test/actions.js

```diff
@@ -62,9 +64,8 @@ describe( 'actions', () => {
 		context( 'failure', () => {
 			useNock( nock => {
 				nock( 'https://public-api.wordpress.com:443' )
-					.persist()
 					.post( '/rest/v1.1/me/block/sites/124/new' )
-					.reply( 500, deepFreeze( { error: 'Server Error' } ) );
+					.reply( 200, deepFreeze( sampleFailureResponse ) );
```

Comment:
A failure with a 200 is usually either:
- you're trying to block a site you've already blocked
- you're trying to unblock a site that you don't have blocked

The endpoint fails quietly in those cases, possibly because there are no ill effects (the outcome is the same as your intended action). If I was building the endpoint from scratch I'd probably throw out a different response code there, but we have to assume that clients rely on that existing behaviour.

I've included the 500 test to make sure we also trigger a block/unblock failure when the endpoint is not working correctly.